data/sql/updates/pending_db_world/missing_npc_loot_1.sql

```diff
@@ -0,0 +1,141 @@
+
+-- Hans Zandin (2396)
+UPDATE `creature_template` SET `lootid` = 2396 WHERE (`entry` = 2396);
+DELETE FROM `creature_loot_template` WHERE (`Entry` = 2396);
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES
+(2396, 4306, 0, 75.0, 0, 1, 0, 2, 2, 'Hans Zandin - Silk Cloth');
+
+-- Fawn (890)
+UPDATE `creature_template` SET `lootid` = 890 WHERE (`entry` = 890);
+DELETE FROM `creature_loot_template` WHERE (`Entry` = 890);
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES
+(890, 2318, 0, 13.33, 0, 1, 0, 1, 1, 'Fawn - Light Leather'),
+(890, 2934, 0, 86.67, 0, 1, 0, 1, 1, 'Fawn - Ruined Leather Scraps');
```

Comment:
Hard doubt on this one. Looks strikingly like skinning loot, not creature loot.

Reply:
I noticed it, we could remove it if we are not sure

Reply:
It and others like it need to be removed. Their source on Wowhead (in the case of 890) lists only 15 samples. This is far too small of a sample size to be legitimately considered IMO.

Reply:
removed loot of 890

Reply:
removed